pkg/account/delete/client_test.go

```diff
@@ -117,8 +118,57 @@ func TestAccountAPIClient_DeleteGroup(t *testing.T) {
 		accountClient := delete.NewAccountAPIClient("1234", accounts.NewClient(restClient))
 
 		err = accountClient.DeleteGroup(t.Context(), "test-group")
-		assert.ErrorIs(t, err, delete.NotFoundErr)
+
+		notFoundErr := &delete.ResourceNotFoundError{}
+		assert.ErrorAs(t, err, &notFoundErr)
 	})
+
+	t.Run("successful delete of service user with no description", func(t *testing.T) {
+		server := httptest.NewServer(http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
```

Comment:
[nit]: maybe it makes sense to have it like in `settings_client_test.go` with mux and handleFun in here and other functionsc. Would be easier to read and we would not need the switch.

Reply:
Added to follow-ups